bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+
+$$
+\begin{align*}
+\dot{\mathbf{p}}_{W B} & =\mathbf{v}_{W B} \\
+\dot{\mathbf{v}}_{W B} & ={ }_{W} \mathbf{g}+\mathbf{q}_{W B} \odot \mathbf{c}  \tag{3}\\
+\dot{\mathbf{q}}_{W B} & =\frac{1}{2} \Lambda\left(\boldsymbol{\Omega}_{B}\right) \cdot \mathbf{q}_{W B}
+\end{align*}
+$$
+where, 
+$\mathbf{p}_{W B}=\left(p_{x}, p_{y}, p_{z}\right)^{\top}$ and $\mathbf{q}_{W B}=\left(q_{w}, q_{x}, q_{y}, q_{z}\right)^{\top}$ are the position and the orientation of the body frame with respect to the world frame $W$, expressed in world frame. Additionally, let $\mathbf{v}_{W B}=\left(v_{x}, v_{y}, v_{z}\right)^{\top}$ be the linear velocity of the body, expressed in world frame, and $\boldsymbol{\Omega}_{B}=\left(\omega_{x}, \omega_{y}, \omega_{z}\right)^{\top}$ its angular velocity, expressed in the body frame. Finally, let $\mathbf{c}=(0,0, c)^{\boldsymbol{\top}}$ be the mass-normalized thrust vector, where $c=\left(f_{1}+f_{2}+f_{3}+f_{4}\right) / m, f_{i}$ is the thrust produced by the i-th motor, and $m$ is the mass of the vehicle. Moreover, where ${ }_{W} \mathbf{g}=(0,0,-g)^{\top}$ is the gravity vector, with $g=$ $9.81 \mathrm{~m} / \mathrm{s}^{2}$. The state and the input vectors of the system are $\mathbf{x}=\left[\mathbf{p}_{W B}, \mathbf{v}_{W B}, \mathbf{q}_{W B}\right]^{\top}$ and $\mathbf{u}=\left[c, \boldsymbol{\Omega}_{B}^{\top}\right]^{\top}$, respectively.
+
+The perception objectives can be formulated such that it guarantees a robust vision-based perception. Let ${ }_{W} \mathbf{p}_{f}=\left({ }_{W} p_{f x}, W p_{f y}, W p_{f z}\right)$ be the 3D position of a point of interest (landmark) in the world frame $W$ (cf. Fig. 22). We assume the body to be equipped with a camera having extrinsic parameters described by a constant rigid body transformation $T_{B C}=\left[\mathbf{p}_{B C}, \mathbf{q}_{B C}\right]$, where $\mathbf{p}_{B C}$ and $\mathbf{q}_{B C}$ are the position and the orientation of $C$ with respect to $B$. The coordinates $C \mathbf{p}_{f}=\left({ }_{C} p_{f x}, C p_{f y}, C p_{f z}\right)^{\top}$ of ${ }_{W} \mathbf{p}_{f}$ in the camera frame $C$ are given by:
+
+$$
+\begin{align*}
+C \mathbf{p}_{f}= & \left(\mathbf{q}_{W B} \mathbf{q}_{B C}\right)^{-1} \odot  \tag{4}\\
+& \left({ }_{W} \mathbf{p}_{f}-\left(\mathbf{q}_{W B} \odot \mathbf{p}_{B C}+\mathbf{p}_{W B}\right)\right)
+\end{align*}
+$$
+
+The point $C \mathbf{p}_{f}$ in camera frame is projected into the image plane coordinates $\mathbf{s}=(u, v)^{\top}$ according to classical pinhole camera model :
+
+$$
+\begin{equation*}
+u=f_{x} \frac{C p_{f x}}{C p_{f z}}, \quad v=f_{y} \frac{C p_{f y}}{C p_{f z}} \tag{5}
+\end{equation*}
+$$
+
+where $f_{x}, f_{y}$ are the focal lengths for pixel rows and columns, respectively.
+Later, these equations can be used to express the projection velocity as a function of the quadrotor state and input vectors as we are interested in reducing the velocity of its projection onto the image plane. 
+
+## Problem Formulation
```

Comment:
This part provides a clear formulation of the optimization problem including objective functions and constraints. It would be really good if the writer can explain how these formulations enhance the quadrotor's ability to navigate or respond to environmental variables.